doc/orchestrator.rst

```diff
@@ -165,3 +165,541 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+  |----|------------------------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 1  | colo_model                   | Model         | 1538905.8   | 0       | 3.5465  | Completed | 0            |
+  | 2  | single_shard_db_identifier_0 | DBNode        | 1538905.5   | 0       | 73.1798 | Cancelled | 0            |
+  | 3  | multi-shard-db-identifier    | DBNode        | 1538905.6+2 | 0       | 49.8503 | Cancelled | 0            |
+
+How to Run the Example
+----------------------
+Below are the steps to run the experiment. Find the
+:ref:`experiment source code<Experiment Source Code>`
+and :ref:`application source code<Application Source Code>`
+below in the respective subsections.
+
+.. note::
+  The example assumes that you have already installed and built
+  SmartSim and SmartRedis. Please refer to Section :ref:`Basic Installation<Basic Installation>`
+  for further details. For simplicity, we assume that you are
+  running on a SLURM-based HPC-platform. Refer to the steps below
+  for more details.
+
+Step 1 : Setup your directory tree
+    Your directory tree should look similar to below::
+
+      SmartSim/
+      SmartRedis/
+      Multi-db-example/
+        application_script.py
+        experiment_script.py
+
+    You can find the application and experiment source code in subsections below.
+
+Step 2 : Install and Build SmartSim
+    This example assumes you have installed SmartSim and SmartRedis in your
+    Python environment. We also assume that you have built SmartSim with
+    the necessary modules for the machine you are running on.
+
+Step 3 : Change the `exe_args` file path
+    When configuring the colocated model in `experiment_script.py`,
+    we pass the file path of `application_script.py` to the `exe_args` argument
+    on line 33 in :ref:`experiment_script.py<Experiment Source Code>`.
+    Edit this argument to the file path of your `application_script.py`
+
+Step 4 : Run the Experiment
+    Finally, run the experiment with ``python experiment_script.py``.
+
+
+Application Source Code
+^^^^^^^^^^^^^^^^^^^^^^^
+.. sourcecode:: python
+  :linenos:
+
+  from smartredis import ConfigOptions, Client
+  from smartredis import *
+  from smartredis.error import *
+
+  single_shard_config = ConfigOptions.create_from_environment("single_shard_db_identifier")
+  app_single_shard_client = Client(single_shard_config, logger_name="Model: single shard logger")
+
+  multi_shard_config = ConfigOptions.create_from_environment("multi-shard-db-identifier")
+  app_multi_shard_client = Client(multi_shard_config, logger_name="Model: multi shard logger")
+
+  colo_config = ConfigOptions.create_from_environment("colo_db_identifier")
+  colo_client = Client(colo_config, logger_name="Model: colocated logger")
+
+  val1 = app_single_shard_client.get_tensor("tensor_1")
+  val2 = app_multi_shard_client.get_tensor("tensor_2")
+
+  app_single_shard_client.log_data(LLInfo, f"The single sharded db tensor is: {val1}")
```

Comment:
I dont know if you want to add some brief comments also in this source code? Or if we include these files in the ss repo (which idk if we're gonna do?) might be nice to have brief comments before each thing that happens like we do in tests eg. " # Start a new Experiment , # Create a database.

Reply:
Im doin all the above!